src/main/java/frc/robot/subsystems/intake/IntakeLoggedInputs.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.subsystems.intake;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class IntakeLoggedInputs implements LoggableInputs {
+    double supMAX;
+    double infMAX;
+    boolean solenoid;
```

Comment:
Make them public and don't shorthand their names

Reply:
done.